packages/shared/src/components/sidebar/sections/ResourceSection.tsx

```diff
@@ -17,6 +27,11 @@ export const ResourceSection = ({
   const changelog = useChangelog();
   const menuItems: SidebarMenuItem[] = useMemo(() => {
     return [
+      {
+        title: 'Advertise',
+        path: businessWebsiteUrl,
+        icon: () => <ListIcon Icon={() => <MegaphoneIcon />} />,
+      },
```

Comment:
```suggestion
      {
        title: 'Advertise',
        path: businessWebsiteUrl,
        icon: () => <ListIcon Icon={() => <MegaphoneIcon />} />,
        target: '_blank',
      },
```

Open in new tab like on mobile?